app/Http/Controllers/DashboardController.php

```
@@ -0,0 +1,365 @@
+            'average_sale_today' => $this->getAverageSale($startOfDay, $endOfDay, $branchFilter),
+            'average_sale_month' => $this->getAverageSale($startOfMonth, $endOfMonth, $branchFilter),
+            'total_products' => $this->getTotalProducts($branchFilter),
+            'low_stock_products' => $this->getLowStockProducts($branchFilter),
+            'total_clients' => $this->getTotalClients($branchFilter),
+            'total_users' => $this->getTotalUsers($branchFilter),
+        ];
+
+
+
+        // Comparación con mes anterior
+        $previousMonthMetrics = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$previousMonthMetrics'. (Unused Code Rules)

(UnusedLocalVariable)